xml/deployment_cluster_administration.xml

```diff
@@ -309,7 +309,27 @@ roleRef:
      <callout arearefs="co.auth.group.role-namespace">
       <para>
        Namespace the new group should be allowed to access. Use
-        <literal>default</literal> for &kube;' default namespace.
+        <literal>default</literal> for &kube;' default namespace
```

Comment:
Enclose the group of callouts in a &lt;calloutlist> if there is none

Reply:
It already is. OK, I will merge it, close the bug and move on. Thank you!